### PR TITLE
Add recipient_wallet_id to GET transfer response

### PR DESCRIPTION
**Pull request by Lerek**

### CODE REVIEW

File: app/views/api/v1/transfers/_transfer.json.jbuilder [hunk 1]
```diff
@@ -2,6 +2,7 @@ json.call(
   transfer.decorate,
   :id,
   :transfer_type_id,
+  :recipient_wallet_id,
```
Comment (aquabu) [inline app/views/api/v1/transfers/_transfer.json.jbuilder:5]: @Lerek seems like there should be a test that changes and the API should be updated.

**Review by aquabu — CHANGES_REQUESTED** [review]

**Review by Riddlerrr — CHANGES_REQUESTED** [review]
What about docs update?

**Comment by Lerek** [comment]
> What about docs update?

@Riddlerrr done, but now there are a lot of conflicts... maybe it makes sense to add `public/doc/api` folder to gitignore and just generate docs on deployment only?

**Comment by Riddlerrr** [comment]
@Lerek Yeah, we thought about it too.
But for now just merge the `acceptance` branch into the current one and re-generate docs ones again.

**Comment by Lerek** [comment]
> @Lerek Yeah, we thought about it too.
> But for now just merge the `acceptance` branch into the current one and re-generate docs ones again.

@Riddlerrr got it, thank you! Done)

**Review by Riddlerrr — APPROVED** [review]

**Review by aquabu — APPROVED** [review]
This looks fine but we are gettin more api doc merge conflicts.

**Comment by Lerek** [comment]
@aquabu done)

**Pull request merged**